ompi/mca/pml/ucx/pml_ucx_request.c

```diff
@@ -282,7 +282,7 @@ void mca_pml_ucx_completed_request_init(ompi_request_t *ompi_req)
     mca_pml_ucx_request_init_common(ompi_req, false, OMPI_REQUEST_ACTIVE,
                                     mca_pml_completed_request_free,
                                     mca_pml_completed_request_cancel);
-    ompi_req->req_mpi_object.comm = &ompi_mpi_comm_world.comm;
+    ompi_req->req_mpi_object.comm = &ompi_mpi_comm_null.comm;
```

Comment:
why is this change?

Reply:
this change was made because its possible that an application is only using the Sessions model for initialization, so MPI_COMM_WORLD is not defined.  MPI_COMM_NULL is defined both when using Sessions or using the world process model.